2018/gulpfile.js

```diff
@@ -12,12 +12,14 @@ const source = require('vinyl-source-stream');
 const buffer = require('vinyl-buffer');
 const uglify = require('gulp-uglify');
 const del = require('del');
+const confCal = require('conf-cal');
```

Comment:
conf-cal is a csv parser?

Reply:
Conf-cal is a parser for the conf-cal textformat parser.

I made the conf-cal concept to make it easy for us to maintain: Here is the raw-data used:

[`conference.confcal`](https://github.com/nodefest/nodefest.github.com/blob/e24664e1e2f08d9cf9bd2d862ff43ebdca61b545/2018/src/confcal/conference.confcal#L26)
[`interactive.confcal`](https://github.com/nodefest/nodefest.github.com/blob/e24664e1e2f08d9cf9bd2d862ff43ebdca61b545/2018/src/confcal/interactive.confcal#L26)